docs/src/app/components/pages/components/LeftNav/ExampleUndocked.jsx

```diff
@@ -3,6 +3,7 @@ import LeftNav from 'material-ui/lib/left-nav';
 import MenuItem from 'material-ui/lib/menus/menu-item';
 import RaisedButton from 'material-ui/lib/raised-button';
 
+
```

Comment:
@tintin1343 You can remove this extra line that was added (or reset your changes to this file).